packages/react-ui-core/src/Gmap/FreeDrawLayer.js

```diff
@@ -125,13 +131,20 @@ export default class FreeDrawLayer extends PureComponent {
 
     this.events = {
       ...setupEvents(map, GMAP_EVENTS, {
-        onMouseMove: e => this.polyline.getPath().push(e.latLng),
-        onTouchStart: e => this.polyline.getPath().push(e.latLng),
+        onMouseDown: () => { this.mouseDown = true },
+        onTouchStart: () => { this.mouseDown = true },
```

Comment:
I’d prefer these two events call one function that sets this value.

Reply:
sure, I'll change that